Bottom: 2163fcc372a3a6d54cfd6ff9386e7e8ee3b7a171
Top:    f630f326ef99cba10e6aae9c014618e9ede076a1
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-05-14 14:17:44 +0800

Refresh of vhost_net-passing-raw-xdp-buff

---

diff --git a/drivers/vhost/net.c b/drivers/vhost/net.c
index aac4d2d..aff338a 100644
--- a/drivers/vhost/net.c
+++ b/drivers/vhost/net.c
@@ -642,7 +642,6 @@ static void handle_tx_copy(struct vhost_net *net)
 			msg.msg_control = &xdp;
 		else
 			msg.msg_control = NULL;
-
 		total_len += len;
 		if (total_len < VHOST_NET_WEIGHT &&
 		    vhost_has_more_pkts(net, vq)) {
